### ISSUE TITLE
Discover our enhanced toolset for developers

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#812]
In GitLab by @kdjstudios on Apr 10, 2018, 10:23

**Submitted by:** "Authorize.Net" <no-reply@mail.authorize.net>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-04-10-71339/conversation

**Server:** NA

**Client/Site:** NA

**Account:** NA

**Issue:** 

![image](/uploads/2bf1a244eb059b50b96bd7081c876bf0/image.png)

Original email: [original_message__5_.html](/uploads/8b88db6c275a5463cf7b69cc32923326/original_message__5_.html)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 26, 2019, 12:35

@smasih @amishra @pchaudhary 

I believe we may close this ticket. As we have made updates API for Authorize over the last year and everything is functioning as expected. If not, please provide an estimate to make sure we are still in compliance.

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Sep 26, 2019, 06:09

@kdjstudios 

Thank you for the update.

There is only one task is pending for Authorize Gateways when communications/transiant error comes they retry only once.

I believe the ticket is http://gitlab.aavaz.biz/AnswerNet/SABilling/issues/1482#note_44181 


Thanks

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 26, 2019, 08:41

@amishra 

Thank you for confirming. Since that task has its own ticket we will close this one.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 26, 2019, 08:41

closed
